openapi2beans/pkg/generator/yaml2java.go

```diff
@@ -51,7 +50,7 @@ func generateDirectories(fs files.FileSystem, storeFilepath string, force bool)
 	if err == nil {
 		if exists {
 			if !force {
-				err = requestDeletionAffirmation(storeFilepath)
+				err = openapi2beans_errors.NewError("generateDirectories: files located in directory requested to to produce beans in: %s", storeFilepath)
```

Comment:
Something more helpful like this?:
"The tool is unable to create files in folder %s because files in that folder already exist. Generating files is a destructive operation, removing all files in that folder prior to new files being created.
If you wish to proceed, delete the files manually, or re-run the tool using the --force option."

Do you agree that would be better for a user to see than "generateDirectories: files located in directory requested to to produce beans in:xxx"

Though if it's a folder that shouldn't already exist the wording would need to change. Wasn't sure as I wrote this.

Reply:
I agree with this :) worded it to make sure it was java file specific though